masterlist.yaml

```diff
@@ -26760,3 +26760,33 @@ plugins:
         util: 'SSEEdit v4.0.4c'
       - crc: 0x76F44BFE
         util: 'SSEEdit v4.0.4c'
+        
+  - name: 'Solitude Docks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/13922/'
+        name: 'Solitude Docks'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/33777/'
+        name: 'Solitude Docks Updated'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64071/'
+        name: 'Rob''s Bug Fixes - Solitude Docks Updated'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - Solitude Docks Updated](https://www.nexusmods.com/skyrimspecialedition/mods/64071/)' ]
+        condition: '(checksum("Solitude Docks.esp", 650E43BB) or checksum("Solitude Docks.esp", A100B5ED) or checksum("Solitude Docks.esp", DDAD84FE) or checksum("Solitude Docks.esp", 5A146CCB))'
+    dirty:
+      - <<: *quickClean
+        crc: 0x650E43BB
+        util: '[SSEEdit v4.0.4c](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 28
+      - <<: *quickClean
+        crc: 0xDDAD84FE
+        util: '[SSEEdit v4.0.4c](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
+        itm: 3
+    clean:
+      - crc: 0xA100B5ED
+        util: 'SSEEdit v4.0.4c'
+      - crc: 0x5A146CCB
+        util: 'SSEEdit v4.0.4c'
+      - crc: 0x6574C43C
+        util: 'SSEEdit v4.0.4c'
```

Comment:
When submitting cleaning data, especially for different versions of a plugin, could you please indicate which checksums correspond to which versions, including version numbers? You can either do so in the commit/PR message or by adding comments labeling each checksum (though don't include version numbers in those, leave those in the commit/PR message). From checking myself, I can see that `0x650E43BB` is the original v1.45, `0xDDAD84FE` is the other replacer v2.1, & `0x6574C43C` is your replacer v2.0.

However, I don't know what the other checksums are. I'm guessing they're the original & the other replacer after QAC (though I get different checksums due to different cleaning settings presumably). If so, they should be removed from the PR. They're not really worth adding since they depend on your cleaning settings & users will know the plugin is clean since they cleaned it themselves. They should be removed from the `patchUpdateAvailable` message condition as well.

Reply:
> I always put these in the order of the URL links.

Right, but this wouldn't work in the case of, e.g., the plugin from the first link being clean & the plugin from second being dirty (the `dirty` key should come before `clean` for stylistic consistency), so it'd be helpful to explicitly list out which checksums correspond to which versions & version numbers regardless going forward.
> And as far as i know there aren't any cleaning settings that could be different.

They're in the **Cleaning** menu in the options:

![image](https://user-images.githubusercontent.com/69141501/202101557-e7007e9e-f9b3-4cc2-b2d9-4b0a847dd5dd.png)
However, they're only for UDRs, so I'm not actually sure why we ended up with different checksums after QAC. Either way, it's all the more reason not to add clean checksums for dirty mods.